content/docs/2.9/scalers/azure-service-bus.md

```diff
@@ -41,6 +41,8 @@ triggers:
 - `subscriptionName` - Name of the Azure Service Bus queue to scale on. (Optional*, Required when `topicName` is specified)
 - `namespace` - Name of the Azure Service Bus namespace that contains your queue or topic. (Optional*, Required when pod identity is used)
 - `connectionFromEnv` - Name of the environment variable your deployment uses to get the connection string of the Azure Service Bus namespace. (Optional)
+- `useRegex` - This parameter allows the usage of regex (in the `queueName` or `subscriptionName` parameters). (Values: `boolean`, Default: `false`, Optional)
```

Comment:
I'd say that in other places we have this
```suggestion
- `useRegex` - This parameter allows the usage of regex (in the `queueName` or `subscriptionName` parameters). (Values: `true`, `false`, Default: `false`, Optional)
```

Reply:
I'll change this. Any other changes besides this?